Source/Scene/WebMapServiceImageryProvider.js

```diff
@@ -113,6 +122,29 @@ define([
 
         var pickFeatureResource = resource.clone();
 
+        var that = this;
+        this._reload = undefined;
+        if (defined(options.times)) {
+            this._timeDynamicImagery = new TimeDynamicImagery({
+                clock : options.clock,
+                times : options.times,
+                requestImageFunction : function(x, y, level, request, interval) {
+                    return requestImage(that, x, y, level, request, interval);
+                },
+                reloadFunction : function() {
+                    if (defined(that._reload)) {
+                        that._reload();
+                    }
+                }
+            });
+        }
+
+        if (defined(options.clock)) {
```

Comment:
Why is the `time` parameter a special case? Can't the dynamic properties remain generic like WMTS?

Reply:
The user can pass any parameters they want, but in this case we are building a time parameter out of the time intervals / clock information.

Reply:
I have made changes, but I still get the current time from the clock and create a time parameter if it does not exist in the parameters from the interval.

Reply:
We shouldn't be hard coding that `time` parameter. When I hooked it up into WMTS, there were many different ways server used this parameter. For instance some of them expected `TIME` as the parameter. Others had a parameter `epoch`, and `time` was the offset from this time or some expected `time` to be a range in the format `start`/`end`. In these cases, this code wouldn't work. We don't do this in WMTS and I don't think we should do it here. This needs to be generic and not specific to your use case.